apps/phone/src/apps/twitter/components/tweet/TweetList.tsx

```diff
@@ -2,20 +2,47 @@ import { memo, useState } from 'react';
 
 import { List } from '@ui/components/List';
 import Tweet from './Tweet';
-import { FormattedTweet } from '@typings/twitter';
+import { FormattedTweet, TwitterEvents, GetTweet } from '@typings/twitter';
 import Backdrop from '@ui/components/Backdrop';
+import InfiniteScroll from 'react-infinite-scroll-component'
+import { ServerPromiseResp } from '@typings/common';
+import fetchNui from '@utils/fetchNui';
+import { processTweet } from '@apps/twitter/utils/tweets';
 
 export function TweetList({ tweets }: { tweets: FormattedTweet[] }) {
   const [imageOpen, setImageOpen] = useState<string | null>(null);
+  
+  const [tweetsData, setTweetsData] = useState(tweets);
+  const [page, setPage] = useState(0)
+  const [hasMore, setHasMore] = useState((tweetsData.length >= 20 ? true : false))
+
+  const loadmore = () => {
```

Comment:
This doesn't solve the original issue the phone had with InfiniteScroll before, reference: https://github.com/project-error/npwd/issues/639#issuecomment-1050162876

Any infinite scrolling that would be added would need to be able to cull elements which no major library currently offers.

Reply:
I see, I'm pretty sure this is easily solvable thinking about it. ill remove it for the meantime and set it back to the hardcoded limit of 25 and revisit this later. I have a pretty good idea how I could easily get around this problem.